samplers/aws/xray/internal/util/timer.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util // import "go.opentelemetry.io/contrib/samplers/aws/xray/internal/util"
```

Comment:
This kind of package name is an anti-pattern.  As much as possible these should be either in the internal package or something like `ticker`.

Reply:
That's a good call out. I can move these files to internal package.